auth/native/data.go

```diff
@@ -4,9 +4,24 @@
 
 package native
 
-import "text/template"
+import (
+	"text/template"
 
-var resetEmailData = template.Must(template.New("reset").Parse(`Subject: [tsuru] Password reset process
+	"github.com/tsuru/config"
+)
+
+
+var passwordChars = "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz1234567890_@#$%^&*()~[]{}?=-+,.<>:;`"
+
+func getEmailResetPasswordTemplate() (*template.Template) {
+	templateFile, _ := config.GetString("email-reset-password-template-file")
```

Comment:
The `error` should not be ignored here. You should change this function to return `Template` and `error`:

```
func getEmailResetPasswordTemplate() (*template.Template, error) {
```